dataactbroker/handlers/dashboard_handler.py

```diff
@@ -338,3 +337,152 @@ def historic_dabs_warning_graphs(filters):
         results[file_type] = [sub_dict for sub_id, sub_dict in file_dict.items()]
 
     return JsonResponse.create(StatusCode.OK, results)
+
+
+def historic_dabs_warning_table(filters, page, limit, sort='period', order='desc'):
+    """ Returns a list of warnings containing all the information needed for the DABS dashboard warning table based
+        on the filters provided that represent one page of the table.
+
+        Args:
+            filters: a dict containing the filters provided by the user
+            page: page number to use in getting the list
+            limit: the number of entries per page
+            sort: the column to order on
+            order: order ascending or descending
+
+        Returns:
+            Limited list of warning metadata and the total number of error metadata entries for the given filters
+    """
+
+    validate_historic_dashboard_filters(filters, graphs=True)
```

Comment:
Should we add validation for the pagination arguments? At the very least it should address the expected values for `sort`. With that, we can address lines 419-420.